### PR TITLE
added general info block to offer details page

### PR DESCRIPTION
**Pull request by Tolik170**
image
image image

### CODE REVIEW

**Review by FryDay969 — APPROVED** [review]

File: src/containers/offer-details/offer-general-info/constants.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  icon: {
+    display: 'flex',
+    alignItems: 'center'
+  },
+  title: {
+    typography: { xs: 'body2', sm: 'body1' }
+  }
+}
+
+export const offerGeneralInfoMock = (offer: Offer) => {
```
Comment (MickhaelSh) [inline src/containers/offer-details/offer-general-info/constants.tsx:27]: Why do you use the mock word? if we have offers created and working properly already

File: src/containers/offer-details/offer-general-info/constants.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      icon: <LanguageIcon />,
+      title: 'offerDetailsPage.generalInfo.languages',
+      description: languages.map((item) => (
+        <IconTitleDescription icon={icon} key={item} sx={style} title={item} />
+      ))
+    },
+    {
+      icon: <SellIcon />,
+      title: 'offerDetailsPage.generalInfo.price',
+      description: `${price} UAH/hour`
```
Comment (MickhaelSh) [inline src/containers/offer-details/offer-general-info/constants.tsx:54]: UAH/hour take from translations

**Review by MickhaelSh — COMMENTED** [review]
don`t forget to add storybook for new component

File: src/containers/offer-details/offer-general-info/constants.tsx (hunk 2 above)
Comment (Tolik170) [reply src/containers/offer-details/offer-general-info/constants.tsx]: fixed

File: src/containers/offer-details/offer-general-info/constants.tsx (hunk 1 above)
Comment (Tolik170) [reply src/containers/offer-details/offer-general-info/constants.tsx]: it is not mock for offer itself, it is mock with icon, title, descr for generalInfo cards

File: src/components/icon-title-description/IconTitleDescription.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  icon,
+  title,
+  description,
+  sx = styles
+}) => {
+  return (
+    <Box sx={sx.container}>
+      <Box sx={sx.icon}>{icon}</Box>
+
+      <Box sx={sx.textWrapper}>
```
[the remark this answers is not in the document]
Comment (Tolik170) [reply src/components/icon-title-description/IconTitleDescription.tsx]: fixedd

File: src/containers/offer-details/offer-general-info/constants.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    columnGap: '5px'
+  },
+  title: {
+    display: 'flex',
+    alignItems: 'center'
+  },
+  description: {
+    typography: { xs: 'body2', sm: 'body1' }
+  }
+}
```
Comment (Yurenko) [inline src/containers/offer-details/offer-general-info/constants.tsx:23]: Should these styles be here?
Move it out.

File: src/containers/offer-details/offer-general-info/constants.tsx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+const icon = <DoneIcon fontSize='small' sx={{ color: 'basic.orientalHerbs' }} />
```
Comment (Yurenko) [inline src/containers/offer-details/offer-general-info/constants.tsx:25]: use SizeEnum for fontSize

File: src/containers/offer-details/offer-general-info/constants.tsx (hunk 1 above)
Comment (Tolik170) [reply src/containers/offer-details/offer-general-info/constants.tsx]: fixed

File: src/containers/offer-details/offer-general-info/constants.tsx (hunk 4 above)
Comment (Tolik170) [reply src/containers/offer-details/offer-general-info/constants.tsx]: fixed

File: src/containers/offer-details/offer-general-info/constants.tsx (hunk 5 above)
Comment (Tolik170) [reply src/containers/offer-details/offer-general-info/constants.tsx]: fixed

File: src/constants/translations/en/common.json [hunk 6]
```diff
@@ -15,7 +15,7 @@
   "confirmButton": "OK",
   "uah": "UAH",
   "hour": "HOUR",
-  "hourSlash": "/ HOUR",
+  "uahSlashHour": "UAH/hour",
```
Comment (MickhaelSh) [inline src/constants/translations/en/common.json:18]: it was used on offerquearecard, have you fix it there?

**Review by maarekua — APPROVED** [review]

**Review by mxrcury — APPROVED** [review]

**Pull request merged**